app/controllers/collections_controller.rb

```diff
@@ -41,6 +41,12 @@ def update
     end
   end
 
+  def fetch
+    FetchJobCreator.run(druid: params[:id])
+    flash[:notice] = 'Fetching jobs for this collection queued.'
```

Comment:
i know it's longer, but `Fetching jobs for this collection have been queued` or `Queued fetching jobs for this collection` somehow parses better for me.

Reply:
Quibble accepted.